django_tidb/schema.py

```diff
@@ -60,3 +60,10 @@ def add_field(self, model, field):
             self.execute(self._create_unique_sql(model, [field]))
         else:
             super().add_field(model, field)
+
+    def table_sql(self, model):
+        sql, params = super().table_sql(model)
+        tidb_auto_id_cache = getattr(model._meta, "tidb_auto_id_cache", None)
+        if tidb_auto_id_cache is not None:
+            sql += " AUTO_ID_CACHE %s" % tidb_auto_id_cache
```

Comment:
What if  `tidb_auto_id_cache` isn't a number?

Reply:
The user will receive an error thrown by django-tidb or TiDB, Django does not provide a way to validate meta option, its built-in option can also result in error when got invalid value.